src/contradictory_claims/cli.py

```diff
@@ -34,7 +41,10 @@ def main():
     drug_lex_path = os.path.join(root_dir, 'input/drugnames/DrugNames.txt')
     virus_lex_path = os.path.join(root_dir, 'input/virus-words/virus_words.txt')
 
-    # Load data
+    # Load and preprocess CORD-19 data
+    covid19_df = filter_metadata_for_covid19(metadata_path, virus_lex_path, pub_date_cutoff)  # noqa: F841
```

Comment:
#noqa: F041 because covid19_df isn't being used yet, I assume?

Reply:
Yes, that's right

Reply:
Yes, that's right